public_html/js/mpform.js

```diff
@@ -0,0 +1,79 @@
+//jQuery time
+var current_fs, next_fs, previous_fs; //fieldsets
+var left, opacity, scale; //fieldset properties which we will animate
+var animating; //flag to prevent quick multi-click glitches
+
+$(".next").click(function(){
+	current_fs = $(this).parent().parent();
```

Comment:
Same issue with namespaces: anchor the DOM select query to the `#pin-*` element id namespace. There might legitimately be any number of `next` classes defined on an arbitrary web page.